source/install/cloud-changelog.md

```diff
@@ -3,13 +3,99 @@
+ - Updated email templates to the new branding.
+
+### Improvements
+
+#### User Interface (UI)
+ - Improved typing performance when the emoji autocomplete is open.
+ - Renamed the “View image” modal to “File preview” modal.
+ - Added “Invite People” to the main "+" button below the hamburger menu.
+ - Dropped support for left-hand side-specific bot icons.
+ - Add a "rest field" to the app command parser.
+ - The whole category bounds are now highlighted while holding a channel above a category name on the left-hand side.
+ - Updated **Account Settings > Display > Timezone** to be more user friendly.
+ - Updated feature discovery images.
+ - New theme agnostic file preview modal takes up the full screen. The file preview now has information about the user, channel, and the file, and moves away from text-based buttons to icon-based buttons.
+ - Removed the Slack importer from the user interface.
+ - Increased the limit of uploaded file attachments per post from 5 to 10.
+ - Removed the legacy channel sidebar.
+ - Added desktop notifications for followed threads.
+ - Hungarian and English-Australian are now official languages.
+ - Added an A/B test for prewritten messages.
+ - Added a query param to translate in-product help pages when opened from desktop app.
+ - Added support for multiselect on apps slash commands.
+ - Added rendering for posts containing markdown in email notifications.
+ - App commands now make a distinction between the central channel and the right-hand side channel.
+
+#### Administration
+ - Upgraded Go to v1.16.7.
+ - Migrated the extraction command to mmctl.
+ - Dropped support for Elasticsearch versions earlier than v7.
+ - Removed a deprecated "Backend" field from the plugin manifest.
+ - Removed the convert channel endpoint to use ``/channels/{channel_id}/privacy`` instead.
+ - Converted the "Executables" field in the plugin manifest to a map.
+ - Removed deprecated ``Posts.ParentId`` in favor of the semantically equivalent ``Posts.RootId``. Also removed ``CommandWebhook.ParentId`` and ``CompliancePost.ParentId`` for the same reason.
+ - Added support for React components in channel header tooltips registered by plugins.
+ - Bulk imports with attached files now log and continue when a file fails to upload instead of halting.
+ - Updated Bleve to v2 to use the scorch index type.
+ - Removed the following deprecated CLI commands:
```

Comment:
Question: Have we historically included CLI changes in the Cloud change log? I ask because Cloud System Admins can't run CLI commands.

Reply:
Removed them from here.